Makefile

```diff
@@ -27,6 +27,10 @@ lint:
 	ruff $(sources)
 	black $(sources) --check --diff
 
+.PHONY: codespell
+codespell:
```

Comment:
this is going to throw my muscle memory for a loop after typing codspeed enough times to get used to not including the 'e' 😄

Reply:
https://twitter.com/samuel_colvin/status/1625468736277815299